packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -979,8 +980,8 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
     }
 
     const schema = response.result.data?.txn.data as SchemaType
-
-    const schemaId = getUnqualifiedSchemaId(did, schema.data.name, schema.data.version)
+    const schemaDid = response.result.data?.txn.metadata.from as string
+    const schemaId = getUnqualifiedSchemaId(schemaDid, schema.data.name, schema.data.version)
 
     return {
       schema: {
```

Comment:
I understand from this code that the actual issuer Id for this schema is the one in the "from" field. So shouldn't we also put this value into the `issuerId` of the returned object?

Reply:
Yes, thank you for the suggestion. I will make the necessary changes.

Reply:
@genaris  I have made the necessary changes